include/util/static_graph.hpp

```diff
@@ -313,8 +313,8 @@ class StaticGraph
     }
 
   protected:
-    NodeIterator number_of_nodes;
-    EdgeIterator number_of_edges;
+    NodeIterator number_of_nodes = 0;
```

Comment:
We had `maybe-uninitialized` disabled, not sure if there was really problem here, but looks suspicious.